examples/docs_beta_snippets/docs_beta_snippets/guides/quality-testing/unit-testing-assets-and-ops/op-resource.py

```diff
@@ -0,0 +1,21 @@
+import mock
+from dagster_aws.s3 import S3FileHandle, S3FileManager
+
+import dagster as dg
+
+
+@dg.op
+def load_file(file_manager: S3FileManager) -> str:
+    return file_manager.read_data(S3FileHandle("bucket", "path.txt"))
+
+
+# highlight-start
+def test_load_file() -> None:
+    mocked_resource = mock.Mock(spec=S3FileManager)
+    mocked_resource.read_data.return_value = "contents"
+
+    assert load_file(mocked_resource) == "contents"
```

Comment:
It might be worth the extra overhead of using kwargs here - in the simple case it's fine to directly pass, I just worry about ordering issues especially when config, resources, and upstreams are used in tandem

Reply:
created a more complicated example at the end that shows using kwargs